pkgs/development/libraries/proj/default.nix

```diff
@@ -62,10 +62,27 @@ stdenv.mkDerivation (finalAttrs: {
 
   doCheck = true;
 
-  passthru.tests = {
-    python = python3.pkgs.pyproj;
-    proj = callPackage ./tests.nix { proj = finalAttrs.finalPackage; };
-  };
+  passthru =
+    let
+      proj = finalAttrs.finalPackage;
+    in
+    {
+      withProjData = gridPackages:
```

Comment:
Don't most packages implement optional add-ons as parameters to `default.nix`? That is, rather than installing `pkgs.foo.overrides.withBar` I'd install `(pkgs.foo.override { withFoo = true; })`.